### PR TITLE
add `url` to `runtime.flow_run`

### PR DESCRIPTION
**Pull request by zzstoatzz**
offer a convenient way to retrieve the flow run urls from the `runtime.flow_run` for logging, notifications etc

### Example
```python
from prefect import flow, runtime

@flow(log_prints=True)
def foo():
   print(runtime.flow_run.api_url)
   print(runtime.flow_run.ui_url)

foo()
```

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `maintenance`, `fix`, `feature`, `enhancement`, `docs`.

### CODE REVIEW

**Review by jamiezieziula — APPROVED** [review]

File: src/prefect/runtime/flow_run.py [hunk 1]
```diff
@@ -248,6 +250,13 @@ def get_parent_deployment_id() -> Dict[str, Any]:
     return parent_flow_run.deployment_id if parent_flow_run else None
 
 
+def get_flow_run_url() -> Optional[str]:
```
Comment (chrisguidry) [inline src/prefect/runtime/flow_run.py:253]: Should we differentiate this as the UI URL (as opposed to the API URL)?

Comment (zzstoatzz) [reply src/prefect/runtime/flow_run.py:253]: hmm yeah i thought about making this `ui_url`

Comment (chrisguidry) [reply src/prefect/runtime/flow_run.py:253]: Maybe we should provide both?  `.api_url` and `.ui_url`?  Both seem useful in different contexts

Comment (zzstoatzz) [reply src/prefect/runtime/flow_run.py]: added in [fdef7bf](https://github.com/PrefectHQ/prefect/pull/11686/commits/fdef7bfcb13820c633642de0285cba7cf5149006)

**Review by chrisguidry — APPROVED** [review]
Aw heck yeah, nice!

**Pull request merged**